python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -1043,9 +1048,13 @@ def _split_table_grad_and_add_send_vars(self, program, pserver_endpoints):
                     index=op_index + 2,
                     type="send",
                     inputs={'X': self.trainer_side_table_grad_list},
-                    outputs={'Out': []},
+                    outputs={
+                        'Out':
+                        [self.grad_name_to_send_dummy_out[self.table_name]]
+                        if self.sync_mode else []
+                    },
                     attrs={
-                        "sync_mode": True,
+                        "sync_mode": False,
```

Comment:
Why modify ```sync_mode ``` to ```False```?

Reply:
In sync mode, all send are False, because send barrier will do a sync.
In async mode, for big table, we can just send it's gradient to pserver and run next step.